docs/release-notes.md

```diff
@@ -7,17 +7,67 @@ weight: 150
 We recommend always using "tagged" versions of Ondat rather than "latest",
 and to perform upgrades only after reading the release notes.
 
-The latest tagged release is `2.6.0`. For
+The latest tagged release is `2.7.0`. For
 installation instructions see our
 [Install](/docs/reference/cluster-operator/install) page.
 
-The latest CLI release is `2.6.0`, available from
+The latest CLI release is `2.7.0`, available from
 [GitHub](https://github.com/storageos/go-cli/releases).
 
 # Upgrading
 
 To upgrade from version 1.x to 2.x, contact Ondat [support](/docs/support) for assistance.
 
+## 2.7.0 - Released 2022-04-06
+
+### New
+
+k8s & Orchestrator Rolling Upgrade
+
+- Kubernetes rolling upgrade for AWS EKS, Google Anthos, Google GKE, Microsoft Azure, Openshift and Rancher. Other platforms will be supported in future releases
+
+Operator
+
+- Skipped duplicated nodes by hostname during sync
```

Comment:
Just speaking personally, I don't know what this means so I wouldn't expect customers to know either